### PR TITLE
Apply accessibility updates to provider management views

### PR DESCRIPTION
**Pull request by naseberry**
#### What
Accessibility and coding standard updates to provider management views.

#### Ticket
[CBO-788](https://dsdmoj.atlassian.net/browse/CBO-788)

#### Why
To make this section of CCCD accessible for all users.

#### How
- Minor accessibility updates
- Improve coding standard

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -2551,3 +2554,9 @@ en:
       comment_hint: "Don't include any personal or sensitive information."
       send: Send
       rating: 'Overall, how satisfied were you with this service?'
+
+
+
+  # global
+  global_no: "No"
```
Comment (jsugarman) [inline config/locales/en.yml]: i'd put these at the top under `en:`
you could also do this as further refactoring to anchor and reuse

```
en: 
  # anchor
  global_yes: &global_yes Yes
  global_no: &global_no No

  # and use
  my_view:
     anwser_yes: *global_yes
     anwser_no: *global_no
```
we should also probably have a standard/convention relating to use of quotes (single quotes are better IMO and are the default if no quotes specified, so no quotes actually needed except for interpolation and escaping chars)

**Review by jsugarman — COMMENTED** [review]
looks good assuming the relevant views have all been visually checked.

I would take the opportunity to tidy up the Yes/No translations as they are used in various places. see my individual comment.

File: config/locales/en.yml (hunk 1 above)
Comment (naseberry) [reply config/locales/en.yml]: 👍

**Review by jsugarman — APPROVED** [review]
👍 LGTM

separate conversation over whether we should rely on yaml's default of not requiring single quotes for strings as they default to being single quoted 
http://blogs.perl.org/users/tinita/2018/03/strings-in-yaml---to-quote-or-not-to-quote.html
```
global_yes: 'Yes'
```
is the same as:
```
global_yes: Yes
```

**Comment by naseberry** [comment]
> 👍 LGTM
> 
> separate conversation over whether we should rely on yaml's default of not requiring single quotes for strings as they default to being single quoted
> http://blogs.perl.org/users/tinita/2018/03/strings-in-yaml---to-quote-or-not-to-quote.html
> 
> ```
> global_yes: 'Yes'
> ```
> 
> is the same as:
> 
> ```
> global_yes: Yes
> ```

I believe _Yes_ and _No_ are reserved, therefore I have to wrap them in quotes.

**Pull request merged**